velox/common/memory/MemoryPool.h

```diff
@@ -206,10 +207,8 @@ class MemoryPool : public std::enable_shared_from_this<MemoryPool> {
   }
 
   /// Invoked to visit the memory pool's direct children, and calls 'visitor' on
-  /// each visited child memory pool with the parent pool's 'poolMutex_' reader
-  /// lock held. The 'visitor' must not access the parent memory pool to avoid
```

Comment:
> with the parent pool's 'poolMutex_' reader lock held.
> The 'visitor' must not access the parent memory pool to avoid ...

This sentence is no longer correct, delete it.